en/deploy/README.md

```diff
@@ -109,6 +109,8 @@ You need to install your Heroku toolbelt which you can find here (you can skip t
 
 > On Windows you also must run the following command to add Git and SSH to your command prompt's `PATH`: `setx PATH "%PATH%;C:\Program Files\Git\bin"`. Restart the command prompt program afterwards to enable the change.
 
+> After restarting your command prompt, don't forget to go to your `djangogirls` folder again and activate your virtualenv! (Hint: `myvenv\Scripts\activate`)
```

Comment:
I think we should link to the relevant part of the tutorial here.

The problem is that the `activate` syntax is different between windows and linx/mac.